var/spack/repos/builtin/packages/mpich/package.py

```diff
@@ -41,11 +42,25 @@ class Mpich(AutotoolsPackage):
     version('3.1.1', '40dc408b1e03cc36d80209baaa2d32b7')
     version('3.1',   '5643dd176499bfb7d25079aaff25f2ec')
     version('3.0.4', '9c5d5d4fe1e17dd12153f40bc5b6dbc0')
+    version('develop', git='git://github.com/pmodels/mpich')
 
     variant('hydra', default=True,  description='Build the hydra process manager')
     variant('pmi',   default=True,  description='Build with PMI support')
     variant('romio', default=True,  description='Enable ROMIO MPI I/O implementation')
-    variant('verbs', default=False, description='Build support for OpenFabrics verbs.')
```

Comment:
In your commit removing this you mention:

> The verbs variant did not work as implemented, and it not well-supported in any case. Just remove it.

While it's great to clean things up, I'd appreciate a more in-depth description. I assume it wasn't causing you trouble since it is off by default.

This was added by @adamjstewart in https://github.com/LLNL/spack/pull/1944 so may be required for that. Could this be a version issue (not supported in later versions)?

Reply:
I will drop this for now until I have more time to investigate.